tensornetwork/linalg/tests/initialization_test.py

```diff
@@ -177,3 +177,18 @@ def inner_zero_test(dtype):
         numpyCheck = backend_obj.zeros(n.shape, dtype=dtype)
         np.testing.assert_allclose(tensor.array, tensorCheck)
         np.testing.assert_allclose(numpyT.array, numpyCheck)
+
+def test_initialize_orthogonal_tensor_wrt_pivot(backend):
+  shape=(5, 10, 3, 2)
+  pivot_axis=1
+  seed = int(time.time())
+  np.random.seed(seed=seed)
+  backend_obj = backends.backend_factory.get_backend(backend)
+  for dtype in dtypes[backend]["rand"]:
+    tnI = tensornetwork.initialize_orthogonal_tensor_wrt_pivot(
+        shape,
+        dtype=dtype,pivot_axis,
+        seed=seed,
+        backend=backend,non_negative_diagonal)
+    npI = backend_obj.initialize_orthogonal_tensor_wrt_pivot(shape, dtype=dtype, pivot_axis, seed=seed,non_negative_diagonal)
+    np.testing.assert_allclose(tnI.array, npI)
```

Comment:
pls replace with a test that checks if the initialized tensor has the desired properties